server/views/components/promo/index.config.yml

```diff
@@ -1,15 +1,132 @@
+    meta: 
+      type: article
+      date:
+    title: Animating the body
+    copy: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eos eius debitis, ad unde laborum assumenda numquam suscipit sint iure itaque aut pariatur, neque omnis vel tenetur sapiente nam iusto quas.
+- name: Series article
+  context:
+    modifiers: ['series', 'underlined']
+    url: http://google.com
+    image:
+      sources:
+      - src: https://wellcomecollection.files.wordpress.com/2017/01/copyright-daniel-regan-2.png?w=1400&h=&crop=1
+        size: '1200'
+      - src: https://wellcomecollection.files.wordpress.com/2017/01/copyright-daniel-regan-2.png?w=1400&h=&crop=1
+        size: '900'
+      - src: https://wellcomecollection.files.wordpress.com/2017/01/copyright-daniel-regan-2.png?w=1400&h=&crop=1
+        size: '450'
+      alt: placeholder image
+    meta: 
+      type: 'electricity: part 02'
+      date:
+    title: Frakenstein's Monster
+    copy: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eos eius debitis, ad unde laborum assumenda numquam suscipit sint iure itaque aut pariatur, neque omnis vel tenetur sapiente nam iusto quas.
+- name: Latest article
+  context:
+    modifiers: ['underlined']
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+      alt: placeholder image
+    meta: 
+      type:
+      date: 2 days ago
+    title: Oops!...I wrote a Britney blog post
+    copy:
+- name: Gallery
+  context:
+    modifiers: ['underlined']
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+      alt: placeholder image
+    meta: 
+      type: gallery
+      date:
+      length: 37
+    title: Science and art
+    copy:
+- name: Podcast
+  context:
+    modifiers: ['underlined']
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+      alt: placeholder image
+    meta: 
+      type: podcast
+      date:
+      length: '56:02'
+    title: The designs of the Festival Pattern Group
+    copy:
+- name: Video
+  modifiers: ['underlined']
+  context:
+    url: http://google.com
+    image:
+      sources:
+      - src: https://placehold.it/1200x900
+        size: '1200'
+      - src: https://placehold.it/900x600
+        size: '900'
+      - src: https://placehold.it/450x300
+        size: '450'
+      alt: placeholder image
+    meta: 
+      type: video
+      date:
+      length: '1:37'
+    title: Tomorrow Belongs to Me
+    copy:
```

Comment:
This is a great example of where using a [JS config](https://github.com/wellcometrust/wellcomecollection.org/pull/355/files#diff-00b52530f56131febb32f260c08eb872R1) would be useful.